<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetChat</class>
 <widget class="QWidget" name="WidgetChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Chat/Chat.png</normaloff>:/Resource/Chat/Chat.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterChat">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frameChatLeftSidebar">
      <property name="maximumSize">
       <size>
        <width>188</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_49">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayoutChatRoomsTask">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0" colspan="2">
          <widget class="QFrame" name="frameChatRoomsTask">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_43">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="3" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayoutChatRoomsButtons">
              <item row="0" column="0">
               <widget class="QToolButton" name="toolButtonChatRoomAdd">
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset resource="../Resource.qrc">
                  <normaloff>:/Resource/Generic/Add.png</normaloff>:/Resource/Generic/Add.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="toolButtonChatRoomRemove">
                <property name="text">
                 <string>Remove</string>
                </property>
                <property name="icon">
                 <iconset resource="../Resource.qrc">
                  <normaloff>:/Resource/Generic/Remove.png</normaloff>:/Resource/Generic/Remove.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QListView" name="listViewChatRooms">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="toolButtonChatRoomsHeader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Rooms</string>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Resource/Chat/Rooms.png</normaloff>:/Resource/Chat/Rooms.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayoutChatFriendsTask">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButtonChatFriendsHeader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Friends</string>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Resource/Chat/Friends.png</normaloff>:/Resource/Chat/Friends.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="frameChatFriendsTask">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_51">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayoutChatFriendsButtons">
              <item row="0" column="0">
               <widget class="QToolButton" name="toolButtonChatFriendAdd">
                <property name="text">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset resource="../Resource.qrc">
                  <normaloff>:/Resource/Generic/Add.png</normaloff>:/Resource/Generic/Add.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="toolButtonChatFriendRemove">
                <property name="text">
                 <string>Remove</string>
                </property>
                <property name="icon">
                 <iconset resource="../Resource.qrc">
                  <normaloff>:/Resource/Generic/Remove.png</normaloff>:/Resource/Generic/Remove.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QListView" name="listViewChatFriends">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayoutChatMiddle">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QFrame" name="frameChatRightSidebar">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_46">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayoutChatUsersTask">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QFrame" name="frameChatUsersTask">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_41">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayoutChatUsersButtons">
              <item row="0" column="0">
               <widget class="QToolButton" name="toolButtonChatAddFriend">
                <property name="text">
                 <string>Add To Friends</string>
                </property>
                <property name="icon">
                 <iconset resource="../Resource.qrc">
                  <normaloff>:/Resource/Generic/Add.png</normaloff>:/Resource/Generic/Add.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="toolButtonChatIgnore">
                <property name="text">
                 <string>Ignore</string>
                </property>
                <property name="icon">
                 <iconset resource="../Resource.qrc">
                  <normaloff>:/Resource/Generic/Ban.png</normaloff>:/Resource/Generic/Ban.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolButtonStyle">
                 <enum>Qt::ToolButtonTextBesideIcon</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelChatUsersCount">
              <property name="text">
               <string>0 Operators, 0 Total</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QListView" name="listViewChatUsers">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="toolButtonChatUsersHeader">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>In Room</string>
           </property>
           <property name="icon">
            <iconset resource="../Resource.qrc">
             <normaloff>:/Resource/Chat/Users.png</normaloff>:/Resource/Chat/Users.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonChatRoomsHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameChatRoomsTask</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonChatFriendsHeader</sender>
   <signal>toggled(bool)</signal>
   <receiver>frameChatFriendsTask</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
